apps/dialtone-documentation/docs/.vuepress/baseComponents/tokens/TokensBar.vue

```diff
@@ -81,12 +111,26 @@ const resetSearch = () => {
 const hasSearchTerm = computed(() => searchInput.value && searchInput.value.trim().length > 0);
 
 const setFormat = (newFormat) => {
+  router.replace({ path: '/tokens/', query: { ...route.query, format: newFormat } });
   emit('changeFormat', newFormat);
 };
 
 const setTheme = (newTheme) => {
+  router.replace({ path: '/tokens/', query: { ...route.query, theme: newTheme } });
   emit('changeTheme', newTheme);
 };
+
+const copyURLToClipboard = async () => {
+  try {
+    const defaultValue = shareLinkTooltip.value;
+    await navigator.clipboard.writeText(window.location);
+    shareLinkTooltip.value = 'Copied';
+    await new Promise(resolve => setTimeout(resolve, 750));
+    shareLinkTooltip.value = defaultValue;
+  } catch (err) {
+    console.error('Error copying to clipboard');
+  }
```

Comment:
Im guessing if there is an error we should notice the user about it, maybe on the tooltip message, sth like:

```
shareLinkTooltip.value = 'Failed to copy';
```

then we put the default text value again

```
 setTimeout(() => {
      shareLinkTooltip.value = defaultValue;
    }, 750);
```